pom.xml

```diff
@@ -447,11 +452,11 @@
         <artifactId>jboss-logging</artifactId>
         <version>3.4.1.Final</version>
       </dependency>
-      <dependency>
```

Comment:
line 450 "<!-- Required to make Hibernate 4 logging compatible with log4j2 -->" - might be able to be removed?

Reply:
Yep, a slightly newer version is now included as a dependency of hibernate core, so this no longer needs to be managed here. Removed and tested - ensured that hibernate messages still appeared in `miso_debug.log`